archive-cores/archive-cores.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+set -euo pipefail
+
+# Archive all cores corresponding to a given set of core_patterns to
+# $artifacts_output_directory/cores.
+# When possible, the binary that generated the core is also archived, and
+# backtraces are generated and archived.
+#
+# Example:
+# ./archive_cores.sh $ARTIFACT_DIR ./core.* /tmp/core.*
+
+if [ "$#" -lt 2 ]; then
+    echo "usage: $0 <artifacts_output_directory> <core_patterns...>"
```

Comment:
usage is wrong, is shows only 1 position arg and 1 key word arg.
or the scrips has 2 positional args, and 1 key word arg.

Reply:
Indeed, I forgot to update it. thanks!